config/semi_automated.yml

```diff
@@ -449,6 +449,18 @@ parameters:
             tag: 'OAuth CID: 860'
             link: User:Enterprisey/YABBR
             namespaces: [0]
+        effp-helper:
+            regex: '\(\[\[User:Suffusion of Yellow\/effp-helper(\.js)?\|effp-helper\]\]\)'
+            link: User:Suffusion of Yellow/effp-helper
+            namespaces: [0]
```

Comment:
This could be used in any namespace, right?

Reply:
Yes. I originally had it in ns:0 only since that is likely where its used most, but I have removed that limitation